templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,34 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
-
-    {% if manufacturer_list %}
-      <table class="table">
+  <h1>Manufacturer list <a style="right: auto " href="{% url "taxi:manufacturer-create" %}">Create new one</a></h1>
```

Comment:
The `style` attribute uses `right: auto`, which might not have the intended effect. Consider using a more appropriate CSS property if you want to position the link.